src/Reflection/RepositoryMethodsClassReflectionExtension.php

```diff
@@ -24,7 +24,7 @@ public function setBroker(Broker $broker): void
     public function hasMethod(ClassReflection $classReflection, string $methodName): bool
     {
         if (
-            !$classReflection->getNativeReflection()->hasMethod($methodName)
+            !$classReflection->getNativeReflection()->hasMethod($methodName)->yes()
```

Comment:
Hi, this is wrong. If you'd run PHPStan itself on this extension, it would point it out. This code will crash. Only `Type::hasMethod()` etc. were changed, but here you're calling method from PHP itself!